internal/manager.go

```diff
@@ -72,12 +72,13 @@ func (m *Manager) EnvKeys(tvs toolset.MultiToolVersions) (*env.Envs, error) {
 		Paths:     env.NewPaths(env.EmptyPaths),
 	}
 
-	tools := make(map[string]string)
+	tools := make(map[string]struct{})
 	for _, t := range tvs {
 		for name, version := range t.Record {
 			if _, ok := tools[name]; ok {
 				continue
 			}
+			tools[name] = struct{}{}
```

Comment:
Good catch. But this fix is incorrect. The main logic here is that I want to use the first available version in MuitlToolVersion, so this line needs to be at the bottom, where we can get the correct EnvKeys for the target SDK.